test/unit/test_devices/test_eos_device.py

```diff
@@ -31,17 +31,17 @@ def test_config(self):
         result = self.device.config(command)
 
         self.assertIsNone(result)
-        self.device.native.config.assert_called_with([command])
+        self.device.native.config.assert_called_with(command)
 
     def test_bad_config(self):
         command = "asdf poknw"
 
-        with self.assertRaisesRegex(CommandError, command):
+        with self.assertRaisesRegex(CommandError, command[0]):
```

Comment:
The file needs to import pytest, and this should use pytest to validate exceptions. There should be 2 test cases for the cases CommandError and CommandListError are raised.

An example of using pytest and using the exceptions attributes can be found here: https://github.com/networktocode/pyntc/blob/4e56dfcede757a393e0d1caaa44c46539d949362/test/unit/test_devices/test_base_netmiko.py#L55-L59